src/components/JurorTalkCategory/JurorTalkCategory.jsx

```diff
@@ -20,14 +20,29 @@ class JurorTalkCategory extends Component {
         });
     }
 
+    toggleDisplay = () => {
+        console.log('toggleDisplay clicked!');
+            let toggleStatus = document.getElementById(this.props.id);
+            if (toggleStatus.style.display === "none") {
+                toggleStatus.style.display = "block";
+            } else {
+                toggleStatus.style.display = "none";
+            }
+          
+    }
+
     render() {
         console.log('this.props for JurorTalkCategory', this.props);
+        let submissionsForThisCategory = this.props.store.submissions
+            .filter(submission => submission.category_id === this.props.topicId)
         return(
             <div>
-                <div className='topicHeader' id={this.props.topicId}>
+                <div className='topicHeader'>
+                    <button className='toggleDisplay' onClick={this.toggleDisplay}>Toggle Display</button>
```

Comment:
Looks like its all functionally working! Just a little styling to do. Remember to make this a upside-down triangle. Material-UI may have some icon buttons for that!